### PR TITLE
FIX: seeg picked by ICA

### PR DESCRIPTION
**Pull request by kingjr**
`seeg` are not picked by the ICA (which can be necessary for isolating line noise)

I had a quick look to see whether I should just make one big PR with all seeg picks, but I prefer dealing with them separately as many cases are not trivial / to be discussed (covariance, mixing, plotting, source modeling).

@dengemann

### CODE REVIEW

File: mne/preprocessing/ica.py [hunk 1]
```diff
@@ -346,9 +347,9 @@ def _fit_raw(self, raw, picks, start, stop, decim, reject, flat, tstep,
             self._reset()
 
         if picks is None:  # just use good data channels
-            picks = pick_types(raw.info, meg=True, eeg=True, eog=False,
-                               ecg=False, misc=False, stim=False,
-                               ref_meg=False, exclude='bads')
+            picks = _pick_data_channels(raw.info, exclude='bads',
```
Comment (dengemann) [inline mne/preprocessing/ica.py]: why do we actually have this one?

Comment (dengemann) [inline mne/preprocessing/ica.py]: ah forget about it. it's ok, misread something.

**Comment by dengemann** [comment]
thanks @kingjr looks good to me!

**Comment by Eric89GXL** [comment]
> covariance

I think this one should be fine

> mixing

You mean ICA? Don't know about this one -- @dengemann should it matter if SEEG are included by default in ICA?

> plotting

Should extend just like other channels

> source modeling

The forward code should already raise an error or just ignore them because it shouldn't have defs, but you'll have to double check.

Make sense?

**Comment by kingjr** [comment]
@Eric89GXL it was more like some examples; I'll probably tackle them along the way while I'm facing the issue.

The plotting isn't always obvious when there are interpolations I think.

**Comment by kingjr** [comment]
it's merged but it didn't pass coverage? Can we revert that or shall I open a new branch?

Also, I just noticed that `seeg` don't get pre-whitened at the moment.

**Comment by Eric89GXL** [comment]
I think the coverage bits were a fluke, I don't see how your changes could have broken it. But if they did yes please fix in a subsequent PR

**Pull request merged**